requirements.txt

```diff
@@ -12,5 +12,6 @@ google-search-results
 playwright
 wikipedia
 huggingface_hub
+faiss
```

Comment:
did you mean for this to be faiss-cpu? faiss itself doesnt show up as a package for me, looks like you can pip install faiss-cpu or faiss-gpu (From README here --https://github.com/facebookresearch/faiss)

Reply:
also you will need to add the following to the requirements (consolidating from chain above):
matplotlib
plotly
scipy
sklearn